src/_cffi_src/openssl/dh.py

```diff
@@ -9,7 +9,20 @@
 """
 
 TYPES = """
-typedef ... DH;
+typedef struct dh_st {
+    /* Prime number (shared) */
+    BIGNUM *p;
+    /* Generator of Z_p (shared) */
+    BIGNUM *g;
+    /* Private DH value x */
+    BIGNUM *priv_key;
+    /* Public DH value g^x */
+    BIGNUM *pub_key;
+    /* X9.42/RFC 2631 */
+    BIGNUM *q;
+    BIGNUM *j;
+    ...;
+} DH;
```

Comment:
Those functions will need to be declared and used. You can see examples in the rsa bindings. Going forward all bindings need to be compatible with 1.1.0 as well as earlier versions.